src/components/Books.js

```diff
@@ -1,15 +1,16 @@
-/* eslint-disable react-hooks/exhaustive-deps */
 import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import AddnewBook from './AddnewBook';
 import Book from './Book';
 import { getBooks } from '../redux/books/books';
+import '../index.css';
 
 const Books = () => {
   const book = useSelector((state) => state.books);
   const dispatch = useDispatch();
   useEffect(() => {
     dispatch(getBooks());
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.